File: Vjudge/Team.cpp
/***************************************
Author: Md Rakibul Hasan
date: 2024-08-12 15:47:14
****************************************/

#include <bits/stdc++.h>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(NULL);

    int t;
    cin >> t;
    int solve = 0;
    while (t--)
    {
        int petya, vasya, tonya;
        cin >> petya >> vasya >> tonya;
        if ((petya + vasya + tonya) >= 2)
        {
            solve++;
        }
    }
    cout << solve << endl;

    return 0;
}

File: Codeforces/team.java
import java.util.Scanner;

public class team {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();
        int solve = 0;

        while(t-- > 0){
            int petya = sc.nextInt();
            int vasya = sc.nextInt();
            int tonya = sc.nextInt();

            if((petya+vasya+tonya )>=2){
                solve++;
            }
        }
        System.out.println(solve);
    }
}
